src/borg/archiver.py

```diff
@@ -1566,8 +1566,13 @@ def do_upgrade(self, args, repository, manifest=None, key=None):
     @with_repository(cache=True, exclusive=True, compatibility=(Manifest.Operation.CHECK,))
     def do_recreate(self, args, repository, manifest, key, cache):
         """Re-create archives"""
-        msg = ("recreate is an experimental feature.\n"
-               "Type 'YES' if you understand this and want to continue: ")
+        if args.recompress == 'never':
+            msg = ("recreate is an experimental feature.\n"
+                   "WARNING: Currently borg will not perform any recompression.\n"
```

Comment:
AFAICS, it would output this, if `--recompress` was not given on the commandline. So somehow telling the obvious.

It also would output this, if `--recompress never` is given, same thing.